README.md

```diff
@@ -66,8 +66,7 @@ Adversarial Nibbler is a data-centric competition that aims to collect a large a
 
 This challenge is aimed at researchers, developers, and practitioners in the field of fairness and development of text-to-image generative AI. The interface is intentionally designed to be simple enough that researchers from non-AI/ML communities can participate. Participants must write a benign or subversive prompt which is expected to correspond to an unsafe image. The evaluation server returns several generated images using DataPerf-managed API licenses, and the participant selects an image (or none) that falls into one of our failure mode categories surrounding stereotypes, culturallyinappropriate, or ethically inappropriate generations, among others.
 
-For full documentation, see [the Adversarial Nibbler documentation](https://dynabench.org/tasks/adversarial-nibbler) on DynaBench.
-
+For full documentation, see [the Adversarial Nibbler readme](https://github.com/CharviRastogi/dataperf/blob/main/dataperf-adversarial-nibbler/README.md).
```

Comment:
Should this be a relative link?

Reply:
By relative link do you mean I should replace all "CharviRastogi"s to "mlcommons"

Reply:
I think these could be fully relative links for the readmes and image links, which do not reference the repository, just the location relative to the current file (and rendering will still work in local offline clones):

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files